src/node_file.cc

```diff
@@ -1194,19 +1138,12 @@ static void Chown(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
 
   int len = args.Length();
-  if (len < 1)
-    return TYPE_ERROR("path required");
-  if (len < 2)
-    return TYPE_ERROR("uid required");
-  if (len < 3)
-    return TYPE_ERROR("gid required");
-  if (!args[1]->IsUint32())
-    return TYPE_ERROR("uid must be an unsigned int");
-  if (!args[2]->IsUint32())
-    return TYPE_ERROR("gid must be an unsigned int");
+  CHECK_GE(len, 3);
+  CHECK(args[1]->IsUint32());
+  CHECK(args[2]->IsUint32());
```

Comment:
`IsInt32` (both lines)

Reply:
`FChown` is not changed in this PR and also gets `Uint32Value()`. Are you sure negative uid and gid are possible?

Reply:
hmm... they are definitely theoretically possible but you're right. I'll keep these as uint32s